dbms/src/Common/SharedLibrary.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <memory>
+#include <string>
```

Comment:
Why we need memory and string headers here?

Reply:
SharedLibrary(const std::string & path);
using SharedLibraryPtr = std::shared_ptr<SharedLibrary>;